src/client/components/menu/Menu.component.js

```diff
@@ -0,0 +1,147 @@
+import React from 'react';
+import { BrowserRouter, Link } from 'react-router-dom';
+import './Menu.styles.css';
+import faUser from '../../assets/images/user-login.png';
+import faHeart from '../../assets/images/favorite-icon.png';
+import faShoppingCart from '../../assets/images/shopping-cart.png';
+
+export const Menu = () => {
```

Comment:
MenuLogin and Menu components are very similar, they only difference is in the user login or logout logic

instead of duplicating the code, using only one component that accepts an `isAuthenticated` props to decide what to display (login/singup or logout)